pkg/segment/tracing/handler/tracehandler.go

```diff
@@ -469,25 +469,23 @@ func redMetricsToJson(redMetrics structs.RedMetrics, service string) ([]byte, er
 }
 
 func DependencyGraphThread() {
-	time.Sleep(1 * time.Minute) // Initial one-minute wait
-	depMatrix := MakeTracesDependancyGraph()
-	writeDependencyMatrix(depMatrix)
-
 	for {
 		now := time.Now()
 		nextHour := now.Truncate(time.Hour).Add(time.Hour)
 		sleepDuration := time.Until(nextHour)
 
 		time.Sleep(sleepDuration)
-		depMatrix = MakeTracesDependancyGraph()
-		writeDependencyMatrix(depMatrix)
+
+		// Calculate startEpoch and endEpoch for the last hour
+		endEpoch := time.Now().UnixNano() / int64(time.Millisecond)
+		startEpoch := time.Now().Add(-time.Hour).UnixNano() / int64(time.Millisecond)
```

Comment:
Just use `UnixMilli()` instead of `UnixNano() / time.Millisecond`

Reply:
done i pushed the changes